packages/app/src/runner/automation/AutInfo.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div
+    class="flex flex-col p-32px gap-16px items-center"
+    :style="{width: `${windowWidth}px`}"
+  >
+    <img
+      class="h-64px w-64px"
+      src="../../../../frontend-shared/src/assets/logos/cypress-dark.png"
+    >
+    <slot />
+  </div>
+</template>
```

Comment:
Seems like the alert overflows this container at smaller sizes; is it worth adding another wrapper element with a min-width/horizontal overflow? 

![Screen Shot 2022-03-29 at 6 15 34 PM](https://user-images.githubusercontent.com/1711637/160722066-3e28d39b-e3bc-4c47-b6c2-a8bedaf2c3f2.png)

Reply:
Addressed in [14becb3](https://github.com/cypress-io/cypress/pull/20692/commits/14becb328f52a9237aba69a257c61fa3be8a5a6e). I was setting the min-width for the Alerts on the `Automation{Disconnected,Missing}.vue` components so it was easy to add.
<img width="500" alt="Screen Shot 2022-03-30 at 9 30 53 AM" src="https://user-images.githubusercontent.com/25158820/160860485-31862465-d0f6-48f7-aafb-0b48844eca4b.png">